packages/cfd/src/Stores/Modules/CFD/cfd-store.js

```diff
@@ -365,14 +378,91 @@ export default class CFDStore extends BaseStore {
     }
 
+    async migrateMT5Accounts(values, actions) {
+        const account_to_migrate = this.root_store.client.mt5_login_list.filter(
+            acc => acc.landing_company_short === Jurisdiction.SVG && !!acc.eligible_to_migrate
+        );
+        const promises = account_to_migrate.map(account => {
+            const { eligible_to_migrate } = account;
+            const [type, shortcode] = Object.entries(eligible_to_migrate)[0];
+            const account_type = {
+                category: 'real',
+                type,
+            };
+            this.migrated_mt5_accounts = [
+                ...this.migrated_mt5_accounts,
+                { login_id: account.login, to_account: { ...(eligible_to_migrate ?? {}) } },
+            ];
+            return this.requestMigrateAccount(values, shortcode, account_type);
+        });
+
+        try {
+            const results = await Promise.all(promises);
+            const has_error = results.find(result => result.error);
+
+            if (!has_error) {
+                actions.setStatus({ success: true });
+                actions.setSubmitting(false);
+                this.setError(false);
+                this.setCFDSuccessDialog(true);
+                await this.getAccountStatus(CFD_PLATFORMS.MT5);
+
+                const mt5_login_list_response = await WS.authorized.mt5LoginList();
+                this.root_store.client.responseMt5LoginList(mt5_login_list_response);
+
+                WS.transferBetweenAccounts();
+                this.root_store.client.responseMT5TradingServers(await WS.tradingServers(CFD_PLATFORMS.MT5));
+            } else {
+                await this.getAccountStatus(CFD_PLATFORMS.MT5);
+                this.clearCFDError();
+                this.setMT5MigrationError(has_error?.error?.message);
+                this.root_store.ui.toggleMT5MigrationModal();
+            }
+        } catch (error) {
+            // At least one request has failed
+            // eslint-disable-next-line no-console
+            console.warn('One or more MT5 migration requests failed:', error);
+        }
+    }
+
+    requestMigrateAccount(values, shortcode, account_type) {
+        const name = this.getName(account_type);
+        const leverage = this.mt5_companies[account_type.category][account_type.type].leverage;
+        const type_request = getAccountTypeFields(account_type);
+        const { address_line_1, address_line_2, address_postcode, address_city, address_state, country_code, phone } =
+            this.root_store.client.account_settings;
+
+        return WS.mt5NewAccount({
+            mainPassword: values.password,
+            email: this.root_store.client.email_address,
+            leverage,
+            name,
+            address: address_line_1 || address_line_2,
+            city: address_city,
+            country: country_code,
+            phone,
+            state: address_state,
+            zipCode: address_postcode,
+            migrate: 1,
```

Comment:
Do we need to always have `migrate: 1` ? Aren't any conditions needed here ? This does not make sense to me.

Reply:
@hirad-deriv For migrating accounts, we need to send migrate: 1. This is how BE determines it is a migration request and not a new mt5 account creation request

Reply:
But we are sending it as hard coded value for new account creation as well which is not recommended. Don't we need any checks here ?

Reply:
 If it is new account creation, we are calling openMT5Account function. The above function is only for migration